ckan/public/base/javascript/modules/follow.js

```diff
@@ -30,6 +30,19 @@ this.ckan.module('follow', function($) {
 			this.el.on('click', this._onClick);
 		},
 
+		/* Adds the X-CSRFToken header
+		*/
+		addCsrfTokenHeader: function() {
```

Comment:
It would be nice to move this function to the base `module.js` class so it's available to all CKAN modules

Reply:
Actually, having tested it again now, this doesn't work at all :)

First of all because the name is wrong, should be:

```
csrftoken = $('meta[name=_csrf_token]').attr('content')
```

Otherwise csrftoken is `undefined`. 
But that doesn't matter anyway because we are exempting all API calls from CSRF protection here:
https://github.com/ckan/ckan/pull/6863/files#diff-97f365b947bcdf4f698327bcdfcb6409e574e9ef13ac899a18e81be73b61d2d4R212

See other comment for discussion about this

Reply:
Yes, I made this function before we decided to exempt the CKAN API actions, and after a while, I changed the csrf_name... and forgot to change it here.
I think there is no harm in having this function, maybe we will have use of it later :) @amercader